packages/demand-api/src/loyalty/types.ts

```diff
@@ -4,20 +4,32 @@ export type ClientId = string // could be a list of supported clients but lib is
 
 export type LoyaltyStatusResponse = {
   balance?: number
-  can_burn_points?: boolean
+  can_burn?: boolean
+  can_earn?: boolean
```

Comment:
Let's mark these 2 flags as required

Reply:
@jaroslaw-gorecki what about balance? IMHO that should be required as well, quite important information

Reply:
No it is optional - for guest users it won't be returned